### PR TITLE
Uptake validation channel name changes to fix official build

### PR DESCRIPTION
**Pull request by dagood**
Fixes https://github.com/dotnet/windowsdesktop/issues/397

In dotnet/runtime we removed this infra to stop getting these breaking changes. This means the readme doesn't get updated, we're depending on https://github.com/dotnet/arcade/issues/3963 to set those up again. Consider doing the same here.

/cc @NikolaMilosavljevic @RussKie @vatsan-madhavan @MichaelSimons

### CODE REVIEW

**Comment by mmitche** [comment]
Yep, I got some time to work on the aka.ms link this week, getting closer.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -89,9 +89,9 @@ stages:
           storage: master
           public: true
 
-      - dependsOn: PVR_Publish
+      - dependsOn: Net_Eng_Validation_Publish
         channel:
-          name: .NET Tools - Validation
-          bar: PublicValidationRelease_30_Channel_Id
+          name: .NET Eng - Validation
+          bar: Net_Eng_Validation_Channel_Id
```
Comment (mmitche) [inline azure-pipelines.yml]: The Eng - Validation channel is supposed to be for tooling. There is a "General Testing" channel for these purposes.

**Comment by dagood** [comment]
For future reference

Variable names come from here:

https://github.com/dotnet/windowsdesktop/blob/322773a3f28de7350775187fc6cc9a302db3100c/eng/common/templates/post-build/common-variables.yml#L16-L18

Job ids come from here:

https://github.com/dotnet/windowsdesktop/blob/322773a3f28de7350775187fc6cc9a302db3100c/eng/common/templates/post-build/post-build.yml#L153-L155

**Comment by RussKie** [comment]
@AdamYoblick

**Pull request merged**